recipes/librdkafka/all/conandata.yml

```diff
@@ -1,12 +1,15 @@
 sources:
+  "2.0.2":
+    url: "https://github.com/edenhill/librdkafka/archive/v2.0.2.tar.gz"
+    sha256: "f321bcb1e015a34114c83cf1aa7b99ee260236aab096b85c003170c90a47ca9d"
   "1.9.2":
-    url: "https://github.com/edenhill/librdkafka/archive/refs/tags/v1.9.2.tar.gz"
+    url: "https://github.com/edenhill/librdkafka/archive/v1.9.2.tar.gz"
     sha256: "3fba157a9f80a0889c982acdd44608be8a46142270a389008b22d921be1198ad"
   "1.9.1":
-    url: "https://github.com/edenhill/librdkafka/archive/refs/tags/v1.9.1.tar.gz"
+    url: "https://github.com/edenhill/librdkafka/archive/v1.9.1.tar.gz"
     sha256: "3a54cf375218977b7af4716ed9738378e37fe400a6c5ddb9d622354ca31fdc79"
   "1.8.2":
-    url: "https://github.com/edenhill/librdkafka/archive/refs/tags/v1.8.2.tar.gz"
+    url: "https://github.com/edenhill/librdkafka/archive/v1.8.2.tar.gz"
```

Comment:
I've already mentioned this point in another PR, but please don't remove `refs/tags`, the opposite should be done actually. See https://github.com/bazel-contrib/SIG-rules-authors/issues/11#issuecomment-1029861300